content/nginxaas-google/getting-started/create-deployment/deploy-console.md

```diff
@@ -18,12 +18,12 @@ Before you can deploy NGINXaaS, follow the steps in the [Prerequisites]({{< ref
 
 ### Create a network attachment
 
-NGINXaaS requires a [network attachment](https://cloud.google.com/vpc/docs/about-network-attachments) to connect your consumer Virtual Private Cloud (VPC) network and your NGINXaaS deployment's VPC network.
+NGINXaaS requires a [network attachment](https://cloud.google.com/vpc/docs/about-network-attachments) to connect your NGINXaaS deployment to your VPC network.
 
 1. Access the [Google Cloud Console](https://console.cloud.google.com/).
 1. Create a consumer VPC network and subnetwork. See [Google's documentation on creating a VPC and subnet](https://cloud.google.com/vpc/docs/create-modify-vpc-networks#console_1) for a step-by-step guide.
-   - The region you choose in this step must match the region where your NGINXaaS deployment will be created.
-1. Create a network attachment in your new subnet that automatically accepts connections. See [Google's documentation on creating a network attachment](https://cloud.google.com/vpc/docs/create-manage-network-attachments#console_1) for a step-by-step guide.
+   - The region you choose in this step **must match** the region where your NGINXaaS deployment will be created.
```

Comment:
nit: do we actually want to soften this a bit here since this is docs for the UI and the UI takes care of setting the deployment region to this network attachment region? (still useful to document, but there's nothing really for the user to worry about here)

```suggestion
   - The region you choose in this step is the region where your NGINXaaS deployment will be created.
```

Reply:
rephrased to highlight this a bit more.

Reply:
i was asking about highlighting it _less_

Reply:
I think the highlight is important as a lot of times a user will create an attachment wherever but then go to the console and find out that it may be in a region that we do not support.

Going to resolve this like we talked offline. We can revisit it if needed 👍